<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeEdit</class>
 <widget class="QMainWindow" name="LeEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LeEdit</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTreeWidget" name="dom">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>DOM</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="onglets">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="ongletCode">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Code</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTextEdit" name="code"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ongletApercu">
       <attribute name="title">
        <string>Apercu</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTextEdit" name="apercu"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTreeWidget" name="hierarchie">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>Hierarchie</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>589</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouveau"/>
    <addaction name="actionOuvrir"/>
    <addaction name="actionEnregistrer"/>
    <addaction name="actionEnregistrer_sous"/>
    <addaction name="actionFermer"/>
   </widget>
   <widget class="QMenu" name="menuEditer">
    <property name="title">
     <string>Editer</string>
    </property>
    <addaction name="actionCopier"/>
    <addaction name="actionColler"/>
    <addaction name="actionCouper"/>
    <addaction name="separator"/>
    <addaction name="actionRechercher"/>
    <addaction name="actionRemplacer"/>
   </widget>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>Affichage</string>
    </property>
    <addaction name="actionDom"/>
    <addaction name="actionHierarchie"/>
    <addaction name="actionApercu"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionA_propos"/>
    <addaction name="actionMise_a_jour"/>
    <addaction name="actionPreference"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEditer"/>
   <addaction name="menuAffichage"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNouveau">
   <property name="text">
    <string>Nouveau</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </action>
  <action name="actionEnregistrer_sous">
   <property name="text">
    <string>Enregistrer sous</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="text">
    <string>Fermer</string>
   </property>
  </action>
  <action name="actionCopier">
   <property name="text">
    <string>Copier</string>
   </property>
  </action>
  <action name="actionColler">
   <property name="text">
    <string>Coller</string>
   </property>
  </action>
  <action name="actionCouper">
   <property name="text">
    <string>Couper</string>
   </property>
  </action>
  <action name="actionRechercher">
   <property name="text">
    <string>Rechercher</string>
   </property>
  </action>
  <action name="actionRemplacer">
   <property name="text">
    <string>Remplacer</string>
   </property>
  </action>
  <action name="actionDom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dom</string>
   </property>
  </action>
  <action name="actionHierarchie">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hierarchie</string>
   </property>
  </action>
  <action name="actionApercu">
   <property name="text">
    <string>Apercu</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionMise_a_jour">
   <property name="text">
    <string>Mise a jour</string>
   </property>
  </action>
  <action name="actionPreference">
   <property name="text">
    <string>Preference</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionCopier</sender>
   <signal>triggered()</signal>
   <receiver>code</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionColler</sender>
   <signal>triggered()</signal>
   <receiver>code</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCouper</sender>
   <signal>triggered()</signal>
   <receiver>code</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDom</sender>
   <signal>triggered(bool)</signal>
   <receiver>dom</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHierarchie</sender>
   <signal>triggered(bool)</signal>
   <receiver>hierarchie</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
